bin/yugabyted

```diff
@@ -2283,5 +2481,140 @@ def retry_op(func, timeout=180):
     raise RuntimeError("Failed after retrying operation for {} secs.".format(
         now - start_time))
 
+
+class SetupEnvBasedCredentials(object):
```

Comment:
minor: rename this as `EnvBasedCredentials`

Reply:
okay. will do